numpy/lib/function_base.py

```diff
@@ -2268,13 +2268,13 @@ def _vectorize_call_with_signature(self, func, args):
 
 
 def _cov_dispatcher(m, y=None, rowvar=None, bias=None, ddof=None,
-                    fweights=None, aweights=None):
+                    fweights=None, aweights=None, dtype=None):
     return (m, y, fweights, aweights)
 
 
 @array_function_dispatch(_cov_dispatcher)
 def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
-        aweights=None):
+        aweights=None, dtype=None):
```

Comment:
Let's make this keyword-only. (other arguments aren't done that way because the syntax didn't exist when this function was written)
```suggestion
        aweights=None, *, dtype=None):
```